modules/ROOT/pages/platform/metrics-integration.adoc

```diff
@@ -749,3 +749,19 @@ It should show overlapping, ever-increasing lines and if one of the lines levels
 | Default aggregation
 m| MAX
 |===
+
+.Cluster Raft Is Leader
+[frame="topbot", stripes=odd, grid="cols", cols="<1,<4"]
+|===
+| Metric name
+m| neo4j_cluster_raft_is_leader
+| Description
+| Is this server the leader? Track this for each Core cluster member. 
```

Comment:
The concept of Core members was deprecated in 5.0. Should it say "Track this for each rafted primary database in the cluster."?

Reply:
Yes, sorry got the description from deprecated metrics reference. will update.

Reply:
🤔 Although this metric is applicable to V4 instances too. I suppose we need a description that encapsulates both V4 and V5 descriptions in one.

Reply:
How about...
```
Is this server the leader? Track this for each rafted primary database(or in case of `v4` instances, Core member) in the cluster. 
It reports 0 if it is not the leader and 1 if it is the leader. The sum of all of these should always be 1. 
However, there are transient periods in which the sum can be more than 1 because more than one member thinks it is the leader. 
Action may be needed if the metric shows 0 for more than 30 seconds.
```

Reply:
How can we resolve the potential confusion related to the word "database"?

I suggest to simplify it:

```
Is this member the leader?
Track this for each rafted member in the cluster. 
It reports 0 if it is not the leader and 1 if it is the leader. 
The sum of all of these should always be 1. 
However, there are transient periods in which the sum can be more than 1 because more than one member thinks it is the leader. 
Action may be needed if the metric shows 0 for more than 30 seconds.
```

Reply:
Makes sense to hide away any specific reference per version as it may be unsupported at somepoint.